lib/off_broadway/kafka/producer.ex

```diff
@@ -1,23 +1,30 @@
 defmodule OffBroadway.Kafka.Producer do
   @moduledoc """
-  Implements the logic to handle incoming messages through
-  the broadway pipeline. Sends messages to the `handle_info/2`
-  and `handle_demand/2` functions based on requests and tracks
-  acknowledgements in state.
+  Defines a Brodway Producer module which receives messages from Kafka
```

Comment:
```suggestion
  Defines a Broadway Producer module which receives messages from Kafka
```

Reply:
Ran spellcheck and fixed a bunch of typos. Should have done that before...